.github/workflows/cibuildwheels.yml

```diff
@@ -98,10 +113,13 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -r requirements-build.txt
+          # This should be in the [build-system] section of pyproject.toml
+          # python -m pip install -r requirements-build.txt
 
       - name: Build sdist
         run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements-build.txt
           python -m build --sdist
 
       - name: Upload sdist package
```

Comment:
There is a bit of nuance to uploading both `wheel` and `sdist` files appropriately. I am not sure how well that would work with the mismatch between `dist` and `wheelhouse` in the previous steps. See the [latest documentation](https://cibuildwheel.pypa.io/en/stable/deliver-to-pypi/#github-actions) on this issue.